docs/reference/bare_metal/c_small_libs.md

```diff
@@ -0,0 +1,57 @@
+# Using code optimised C libraries in Mbed OS
```

Comment:
`code optimised` i'm not sure what does it mean, `small` or `minimal`?

Reply:
Changed to `small` but I am not convinced it is better.